tortoise/api.py

```diff
@@ -213,7 +213,9 @@ def __init__(self, autoregressive_batch_size=None, models_dir=MODELS_DIR, enable
         self.models_dir = models_dir
         self.autoregressive_batch_size = pick_best_batch_size_for_gpu() if autoregressive_batch_size is None else autoregressive_batch_size
         self.enable_redaction = enable_redaction
-        self.device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+        if device is None:
+            device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
```

Comment:
Hey thanks for the PR and sorry for the late review. Given how poorly this program performs on CPU, can you please print out a warning here?

Reply:
Hey, late review? Guess you didn't expect an update from me.

What do you think about the change?